tests/linter_tests/test_usfm_linter.py

```diff
@@ -98,20 +104,27 @@ def test_PhpIdInvalidCode(self):
         self.replace_tag(out_dir, self.php_file_name, 'id', '\id PH Unlocked Literal Bible')
         expected_warnings = True
         linter = self.run_linter(out_dir)
+        self.assertEqual( linter.log.warnings[0], u'PHP  PH Unlocked Literal Bible -  - Invalid ID' )
+        self.assertEqual( linter.log.warnings[4], u'PHP  04 -  04 - Missing ID before chapter' )
+        self.assertEqual( linter.log.warnings[5], u'Failed to verify book \'51-PHP.usfm\', exception: \'list\' object has no attribute \'push\'' )
         self.verify_results(expected_warnings, linter)
 
     def test_PhpInvalidIdLength(self):
         out_dir = self.copy_resource(self.php_repo_path)
         self.replace_tag(out_dir, self.php_file_name, 'id', '\id PH')
         expected_warnings = True
         linter = self.run_linter(out_dir)
+        self.assertEqual( linter.log.warnings[0], u'PHP  PH -  - Invalid ID' )
```

Comment:
We don't put u'' in front of our strings since we declare unicode a the top of every file.  Please remove all u prefix to strings.

Reply:
Also, please remove spaces after ( and before ). We need to stick to standard. I keep having to go through removing these spaces, as it also keep being flagged by Intellij's linter.